File: mobile/iOS/GuestJini/GuestJini/Global/Model/Responses/KBReviewResponse.swift
//
//  KBReviewResponse.swift
//  GuestJini
//
//  Created by Prasanna Kumar Pete on 22/01/20.
//  Copyright © 2020 Prasanna Kumar Pete. All rights reserved.
//

import Foundation
class KBReviewResponse: Decodable {
    public var kbReview:KBReview? = nil
    public var kbReviewList:[KBReview]?  = nil
    public var totalPages:Int = 0
    public var totalRecords:Int = 0
    public var currentRecords:Int = 0
    public var success:Bool = false
    public var error:String = ""
}

File: mobile/android/GuestJini/app/src/main/java/com/carmel/guestjini/Networking/KnowledgeBase/KBReviewResponse.java
package com.carmel.guestjini.Networking.KnowledgeBase;

import java.util.List;

public class KBReviewResponse {
    private KBReview kbReview;
    private List<KBReview> kbReviewList;
    private int totalPages;
    private int totalRecords;
    private int currentRecords;
    private boolean success;
    private String error;

    public KBReview getKbReview() {
        return kbReview;
    }

    public void setKbReview(KBReview kbReview) {
        this.kbReview = kbReview;
    }

    public List<KBReview> getKbReviewList() {
        return kbReviewList;
    }

    public void setKbReviewList(List<KBReview> kbReviewList) {
        this.kbReviewList = kbReviewList;
    }

    public int getTotalPages() {
        return totalPages;
    }

    public void setTotalPages(int totalPages) {
        this.totalPages = totalPages;
    }

    public int getTotalRecords() {
        return totalRecords;
    }

    public void setTotalRecords(int totalRecords) {
        this.totalRecords = totalRecords;
    }

    public int getCurrentRecords() {
        return currentRecords;
    }

    public void setCurrentRecords(int currentRecords) {
        this.currentRecords = currentRecords;
    }

    public boolean isSuccess() {
        return success;
    }

    public void setSuccess(boolean success) {
        this.success = success;
    }

    public String getError() {
        return error;
    }

    public void setError(String error) {
        this.error = error;
    }
}
